Sources/Starknet/Crypto/StarknetCurve.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+import CryptoToolkit
+import BigInt
+
+public enum StarknetCurveError: Error {
+    case deserializationError
+    case verifyError
+    case unknownError
+}
+
+public class StarknetCurve {
+    private static let curveOrder = BigUInt("800000000000010FFFFFFFFFFFFFFFFB781126DCAE7B2321E66A241ADC64D2F", radix: 16)!
```

Comment:
Can we make this public? In the jvm it is public and we need it to do the [account derivation](https://community.starknet.io/t/account-keys-and-addresses-derivation-standard/1230), more specifically to implement the grinding of the key.